app/controllers/registration_controller.rb

```diff
@@ -103,8 +103,19 @@ def rights_list
   end
 
   def autocomplete
-    response = Dor::SearchService.query('*:*', :rows => 0, :facets => { :fields => [params[:field]] }, :'facet.prefix' => params[:term].titlecase, :'facet.mincount' => 1, :'facet.limit' => 15 )
-    result = response.facets.find { |f| f.name == params[:field] }.items.collect(&:value).sort
+    facet_field = params[:field]
+    facet_fields = { fields: [facet_field] }
+    response = Dor::SearchService.query(
+      '*:*',
+      {
+        rows: 0,
+        facets: facet_fields,
+        :'facet.prefix' => params[:term].titlecase,
+        :'facet.mincount' => 1,
+        :'facet.limit' => 15
+      }
+    )
+    result = response.facets.find { |f| f.name == facet_field }.items.map(&:value).sort
```

Comment:
See also https://github.com/bbatsov/rubocop/issues/1334, where they propose to disable this.